packages/components/src/Payments/PaymentsStyledComponents.tsx

```diff
@@ -285,24 +307,21 @@ export const PaymentsModalCloseBtn = styled(InvertedButton)`
       fill: ${colors.accent.CIVIL_BLUE};
     }
   }
+
+  ${props => props.theme.renderContext === RENDER_CONTEXT.EMBED && "display: none;"}
```

Comment:
Don't see why we would want to prevent users from closing out of payments modal in the embed, I assume this is from the designs? If yes let's ping Rob to confirm.

Reply:
@tobek yes this is from the designs. There is a login avatar component in this spot. There's also a new back button that can take you out of the payments but you would have to keep hitting it depending on how far you got. Maybe there is something else we can do but would be good to play with it first.